content/docs/integrations/agic.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Using Azure Application Gateway Ingress Controller with OSM"
```

Comment:
Documenting this seems unnecessary because this is already well documented in https://docs.openservicemesh.io/docs/tasks_usage/traffic_management/demos/azure_application_gateway_ingress_demo/